CHANGELOG.md

```diff
@@ -29,6 +29,11 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Ben Asher](https://github.com/benasher44)
   [#5959](https://github.com/CocoaPods/CocoaPods/pull/5959)
 
+* Update EMBEDDED_CONTENT_CONTAINS_SWIFT flag behaviour based on xcode version.
+  [codymoorhouse](https://github.com/codymoorhouse)
+  [Ben Asher](https://github.com/benasher44)
```

Comment:
I'm helping with the review, but this is your change! No need to include me, but I appreciate the sentiment :)